src/components/DrinkDetails/DrinkDetails.js

```diff
@@ -1,97 +1,95 @@
-                data[`strIngredient${i}`] !== ""
-            ) {
-                ingredients.push({
-                    measure: data[`strMeasure${i}`],
-                    ingredient: data[`strIngredient${i}`],
-                });
-            }
-        }
+  fetchDrinkData = async () => {
+    let data = await axios.get(
+      '/v1/1/lookup.php?i=' + this.props.match.params.id,
+    );
+    console.log('[fetchDrinkData]', data.data.drinks[0]);
+    data = data.data.drinks[0];
 
-        this.setState({
-            ingredients: ingredients,
-            strInstructions: data.strInstructions,
-            strDrink: data.strDrink,
-            strGlass: data.strGlass,
-            strDrinkThumb: data.strDrinkThumb,
+    let ingredients = [];
+    for (let i = 1; i <= 15; i++) {
+      if (
+        data[`strIngredient${i}`] !== null &&
+        data[`strIngredient${i}`] !== ''
+      ) {
+        ingredients.push({
+          measure: data[`strMeasure${i}`],
+          ingredient: data[`strIngredient${i}`],
         });
-        console.log(this.state);
-    };
+      }
+    }
 
-    render() {
-        let drinkDetails = <h2>Loading, one sec!</h2>;
-        if (this.state.ingredients.length > 0) {
-            let ingredientList = this.state.ingredients.map((ing) => {
-                return (
-                    <li>
-                        <p className={classes.ListItem}>
-                            <strong>{`${ing.measure}`}</strong>
-                            {` - ${ing.ingredient}`}
-                        </p>
-                    </li>
-                );
-            });
+    this.setState({
+      ingredients: ingredients,
+      strInstructions: data.strInstructions,
+      strDrink: data.strDrink,
+      strGlass: data.strGlass,
+      strDrinkThumb: data.strDrinkThumb,
+    });
+    console.log(this.state);
+  };
 
-            drinkDetails = (
-                <Aux>
-                    <h2 className={classes.Title}>{this.state.strDrink}</h2>
-                    <img
-                        className={classes.Image}
-                        src={this.state.strDrinkThumb}
-                        alt={this.state.strDrink}
-                    />
-                    <div className={classes.Instructions}>
-                        <h3 className={classes.SubTitle}>Instructions</h3>
-                        <p>{this.state.strInstructions}</p>
-                    </div>
-                    <div className={classes.Ingredients}>
-                        <h3 className={classes.SubTitle}>Ingredients</h3>
-                        <ul className={classes.IngredientsContainer}>
-                            {ingredientList}
-                        </ul>
-                    </div>
-                </Aux>
-            );
-        }
+  render() {
+    let drinkDetails = <h2 id="loading">Loading, one sec!</h2>;
+    if (this.state.ingredients.length > 0) {
+      let ingredientList = this.state.ingredients.map((ing) => {
```

Comment:
Could we change the `ing` because it makes it harder to read the code to something like `instruction' or something like that 🤔 
Then you will have to change it  on line 61 and 62

Reply:
Just seen the change that makes it easier to read